sherpa/astro/tests/test_astro.py

```diff
@@ -465,12 +465,25 @@ def test_grouped_ciao4_5(parallel, run_thread, clean_astro_ui):
 
     def cmp_thread(fres, aa, covarerr):
         assert fres.numpoints == 46
-        assert fres.statval == approx(18.8316, rel=1e-4)
 
-        assert covarerr[0] == approx(0.104838, rel=1e-4)
-        assert covarerr[1] == approx(2.43937e-05, rel=1e-4)
-        assert aa.gamma.val == approx(1.83906, rel=1e-4)
-        assert aa.ampl.val == approx(0.000301258, rel=1e-4)
+        # Prior to 4.17.0 the selected channel range of 41 to 167
+        # mapped to 1.159:4.79661 keV and resulted in a best-fit
+        # statistic of 18.8316, gamma = 1.83906, and ampl of
+        # 3.01258e-4. In 4.17 the channels now start at 0 so
+        # the energy range has shifted. Ideally I'd change the
+        # test suite, but that is much harder to do than just
+        # change the fit results.
+        #
+        # The new energy range - as of 4.17 - os 1.18764:5.05439
+        # kev. There is not a huge lot of difference between these
+        # results (i.e between these two ranges).
+        #
```

Comment:
I would remove this comment. It makes sense now for me during the review process, but won't make any sense to a reader in the future. Instead, I suggest to put it as a comment the commit message or into this PR on GitHub. Either way, we'll be able to see it for a while (forever if it's in git, but it's a bit harder to associate with the code lines then in the GH web interface.).
Or add another commit that removes this comment - then we have it in the git history for this file.

A reader in the future only needs to know that the test tests what it tests and that any change compared to that is a regression that needs to be investigated.

Reply:
I've just added a commit that removes this text (rather than mess around with a rebase).